webpack.config.js

```diff
@@ -65,7 +65,9 @@ module.exports = (env = {}) => {
       libraryExport: 'default',
       path: Path.resolve(versionDir)
     },
-
+    optimization: {
+      minimize: false
```

Comment:
I feel a little ignorant, but is this an intentional change or was it just for development?

Reply:
This was just for dev, good catch.